internal/xds/translator/runner/runner.go

```diff
@@ -53,7 +55,7 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 				r.Xds.Delete(key)
 			} else {
 				// Translate to xds resources
-				t := &translator.Translator{}
+				t := &translator.Translator{ExtensionManager: &r.ExtensionManager}
```

Comment:
can we follow the code style similar to L60, where we set the value if its not nil ?